src/lib/components/tablev2/Tablev2.component.tsx

```diff
@@ -163,6 +165,14 @@ function Table<
     }
   }, [globalFilter, setGlobalFilter, data]);
 
+  useEffect(() => {
+    if (!allFilters) {
+      setAllFilters([]);
+    } else {
+      setAllFilters(allFilters);
+    }
+  }, [allFilters, setAllFilters]);
```

Comment:
We need to warn that allFilters props need to be stable across re-renders, or ensure it's stability within the context of this component